src/ofono/provider_ofono.hpp

```diff
@@ -120,7 +143,7 @@ class Bridge : public QObject, public PropertiesSource
     typedef std::function<void(Bridge*, QVariant const&)> property_action_type;
     typedef std::map<QString, property_action_type> property_map_type;
 
-    enum class Status { First_ = 0, Offline = First_
+    enum class Status { First_ = 0, Disabled = First_, Offline
```

Comment:
Also, there is a doxygen comment to the ```property_names``` in the provider_ofono.cpp file. RegistrationStatus and Status enums docs should be also updated there.